python/cuml/preprocessing/model_selection.py

```diff
@@ -155,22 +158,33 @@ def train_test_split(
 
     x_numba = False
     y_numba = False
+
+    if seed is not None:
+        if random_state is None:
+            warnings.warn("Parameter 'seed' is deprecatedm please use \
```

Comment:
stray m on deprecated

Reply:
Was `deprecatedm` intended to be `deprecated,`? The sentence does flow better with the comma.

Reply:
it was supposed to be a `,` indeed :P